examples/laravel-image-upload-widget/routes/web.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('/', function () {
+    return view('welcome');
+});
+
+
+Route::get('/do', function () {
+
+   // return Cloudinary::getImage('l56d0bz2hwkntrg09123.png');
```

Comment:
hey what file is this referencing? i wouldnt expect this would work on anyone elses cloud right? was the /do path just for testing?

Reply:
This defines a route for the "URL/do". When it is accessed it retrieves an image and returns its URL. The commented out part was me checking if it was working.